src/dtoValidation/error/Error.java

```diff
@@ -0,0 +1,10 @@
+package dtoValidation.error;
+
+public enum Error {
+    MinLengthError,
```

Comment:
Hakob jan, I think there is not a good place to use enums, we just need to return error messages